addon/modifiers/create-ref.js

```diff
@@ -12,9 +12,11 @@ export default class RefModifier extends Modifier {
     super(...arguments);
     setGlobalRef(getOwner(this));
   }
-  mutationObserverOptions = {
-    attributes: true,
-    characterData: true,
+  // to minimise overhead, user should be specific about
+  // what they want to observe
+  defaultMutationObserverOptions = {
+    attributes: false,
+    characterData: false,
```

Comment:
looks like it's overlapping with https://github.com/lifeart/ember-ref-bucket/pull/14/files

Reply:
Let me know when you've merged it and I'll merge up